templates/zerver/api/upload-custom-emoji.md

```diff
@@ -0,0 +1,46 @@
+# Upload a custom emoji
+This endpoint is used to upload a custom emoji the user's realm
+ `POST {{ api_url }}/v1/realm/emoji/<emoji_name>`
+## Usage examples
+
+{start_tabs}
+
+{tab|python}
+
+{generate_code_example(python)|/realm/emoji/<emoji_name>:post|example}
+
+{tab|curl}
+
+```
+curl {{ api_url }}/v1/realm/emoji/<emoji_name> \
+    -F "data=@/path/to/img.png" \
+    -u USER_EMAIL:API_KEY
+```
+
+{end_tabs}
+
+
+## Arguments
+This endpoint does not consume any arguments.
+
+## Emoji name
+
+can only contain `a-z`, `0-9`, dashes (`-`), and spaces.
```

Comment:
I would write a proper sentence here.

Reply:
Sorry, I'm a bit confused here. Something like `The emoji name can only contain letters, numbers, dashes, and spaces.` is good enough?

Reply:
Yeah, the key part is starting with a capital letter and not trying to use the title as part of the sentence (we don't do that anywhere else).